components/ProductDetails/ProductDetailsOptionalTours.vue

```diff
@@ -146,14 +148,47 @@ h3 {
     cursor: pointer;
 }
 
-.modal-body img {
+.optional-tour .modal-header,
+.optional-tour .modal-footer .btn-secondary,
+.extensions .modal-header,
+.extensions .modal-footer .btn-secondary {
+    display: none !important;
+    height: 0;
+}
+
+.extensions .modal-footer,
+.optional-tour .modal-footer {
+    border: none;
+    justify-content: center;
+}
+
+.optional-tour .modal-content,
+.optional-tour .modal-dialog,
+.extensions .modal-content,
+.extensions .modal-dialog {
+    border: none;
+    margin-top: 0;
+    margin-bottom: 0;
+    max-height: 60vh;
+}
+
+.optional-tour .modal-body,
+.extensions .modal-body {
+    padding-top: 0px !important;
+    padding-bottom: 0px !important;
+}
+
+.optional-tour .modal-body img,
+.extensions .modal-body img {
     height: 220px;
```

Comment:
Would this height work in mobile?

Reply:
I have added this restriction to the media queries, so it only effects large screens.